### PR TITLE
Fix Legrand 067755 adapter

### PR DESCRIPTION
**Pull request by jiquem**
The previous adapter was not able to be used with the 067755.

### CODE REVIEW

File: src/devices/legrand.js [hunk 1]
```diff
@@ -46,11 +46,10 @@ const fzlocal = {
 
 module.exports = [
     {
-        zigbeeModel: [' Pocket remote\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000'+
-            '\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000'],
+        fingerprint: [{modelID: ' Wireless Scenes Command\u0000\u0000\u0000\u0000\u0000\u0000\u0000'}],
```
Comment (Koenkk) [inline src/devices/legrand.js:49]: This removes support for the `Pocket remote`, please add `Wireless Scenes Command` to it.

Comment (jiquem) [reply src/devices/legrand.js:49]: Sorry, I understand that this remove the support for Pocket Remote, but I don't get how to add Wireless Scenes Command to it. Could you please explain me ?

Comment (Koenkk) [reply src/devices/legrand.js:49]: Can you confirm your device also has model `067755` written on it?

Comment (jiquem) [reply src/devices/legrand.js:49]: Yes, it's written on a sticker on it

**Comment by Koenkk** [comment]
thanks!

File: src/devices/legrand.js (hunk 1 above)
Comment (jiquem) [reply src/devices/legrand.js]: Thanks for the modification.

**Pull request merged**